book/website/ethical-research/data-hazards/dh-intro.md

```diff
@@ -0,0 +1,109 @@
+
+Each label has, as described in the main website: 
+
+- **Hazard image**, title, and description which represents and describes the risk.
+- **Examples** to clarify what the hazard covers.
+- **Safety Precautions** things that we would want to see done before the research is deployed.
+
+### Example label: high environmental cost
+```{figure} ../../figures/data-hazard-environmental-cost.*
+---
+height: 250px
+name: data-hazard-environmental-cost
+alt: A red diamond with an image of the earth at the centre. The earth is surrounded by a flame. 
+
+---
+_Data Hazards_ project illustration by Yasmin Dwiputri. Used under a CC-BY 4.0 licence. [DOI.](10.31219/osf.io/hzmyp).
+```
+
+#### Description
+This hazard is appropriate where methodologies are energy-hungry, data-hungry (requiring more and more computation), or use special hardware that requires rare materials. Alternatively, this could also apply to research which outputs high amounts of waste.
+
+#### Examples
+ - Example 1: Wet-lab experiments which create large amounts of single-use plastic waste, for example, pipette tips or petri dishes; or specific chemicals that are harmful to the environment {cite:ps}`Freeland2022`, {cite:ps}`Leak2023`. 
+ 
+ - Example 2: Running big models on High-Performance Clusters can have a big environmental impact with the energy they consume {cite:ps}`alaparthi2020bidirectional`, {cite:ps}`Bender2021`.
+
+#### Safety Precautions
+ - Consider recycling programmes, as well as reducing purchasing and shipping. 
+ - To communicate the scale of the issue to other stakeholders, you may want to [convert units of energy into more relatable units](https://calculator.green-algorithms.org/).
+ - Find out if your cloud provider uses renewable energy.
+ - Consider profiling your code, and rewriting it to use less energy. 
+ - Consider future work that would reduce the need to use increasingly more resources.
+
+---
+
+### But Why Hazard Labels?
+The Data Hazards framework takes inspiration from the visual nature of chemical hazard symbols, communicating to users the potential outcomes and risks of a project. 
+Similarly, to chemical hazards, labelling something as hazardous does not change or reduce its inherent risk.
+However, the visual display of the risk allows the user to make choices based on this knowledge. 
+The user can choose to take the necessary precautions to keep themselves, others and their environment safe. 
+This also applies to the Data Hazard labels, where if we can identify the risks, we can act accordingly and mitigate possible risks.
+
+
+### Where did they come from?
+There were originally six labels, which were identified through reading data ethics papers at [Data Ethics Club](http://dataethicsclub.com): a fortnightly journal club for data science and ethics.
+Through collecting suggestions [on GitHub](), and at conferences and Data Hazards workshops, further labels were suggested and added until we reached the current 11 labels.
```

Comment:
Maybe "the current 11 labels." needs to be contextualised "as of November 2023", or something similar?